sdlib/d/gc/extent.d

```diff
@@ -105,9 +105,15 @@ private:
 		shared Bitmap!256 freeSpaceFlags;
 	}
 
+	struct FinalizationData {
+		ubyte[48] pad;
+		shared Bitmap!128 finalizationFlags;
+	}
+
```

Comment:
You can merge the two structs like this:

[128 bits padding][128 bits finalizers][256 bits freeSpaceFlags]